examples/step-49/doc/intro.dox

```diff
@@ -6,6 +6,10 @@ were contributed by Yuhan Zhou, Wolfgang Bangerth, and David Wells.</i>
 This tutorial is an extension to step-1 and demonstrates several ways to
 obtain more involved meshes than the ones shown there.
 
+@note This tutorial is also available as a Jupyter Python notebook that 
+  uses the deal.II python interface. The notebook is available in the 
+  same directory as the original C++ program.
```

Comment:
Can you move the Python tutorial with the others Python tutorials instead? I think that's what we decided last time.

Reply:
Shall follow the *step-62* and add a symbolic link to the original notebook in the contrib dir? This way I can use the files from the step-49's directory (e.g. example.msh).

Reply:
Yes that's fine too